src/pages/NotFoundPage/NotFound.tsx

```diff
@@ -8,12 +8,12 @@ import {
 } from '@patternfly/react-core';
 import { WrenchIcon } from '@patternfly/react-icons';
 import { Button } from '@patternfly/react-core';
-import { useHistory } from 'react-router-dom';
+import { useNavigate } from 'react-router-dom';
 
 const NotFound: React.FC = () => {
-  const history = useHistory();
+  const navigate = useNavigate();
   function handleClick() {
-    history.push('/');
+    navigate('/');
```

Comment:
Does this go back to the console dot home page or /insights/subscriptions/inventory?

Reply:
/insights/subscriptions/inventory